ospfd/ospf_ri.c

```diff
@@ -1228,10 +1228,10 @@ static int ospf_router_info_lsa_update(struct ospf_lsa *lsa)
 	do {                                                                   \
 		if (ntohs(tlvh->length) > size) {                              \
 			if (vty != NULL)                                       \
-				vty_out(vty, "  Wrong %s TLV size: %d(%d)\n",  \
+				vty_out(vty, "  Wrong %s TLV size: %d(%d)",    \
```

Comment:
We shouldn't remove `\n` from the vty output.

Reply:
yep you are correct, fixing